### PR TITLE
Java manual instrumentation: add another example under context propagation

### PR DESCRIPTION
**Pull request by punidramesh**
The current docs lacks an example that demonstrates the extraction of context from the incoming request, adding some span data, and the continuation of context via the outgoing request. `Httpheaders` and a TextMapGetter implementation have been utilized to extract the `traceparent` header.

### CODE REVIEW

File: content/en/docs/instrumentation/java/manual.md
Comment (cartermp) [inline content/en/docs/instrumentation/java/manual.md:412]: There's a `try...finally` nested in here for an inner scope, but none for this outer scope. Is that intentional?

File: content/en/docs/instrumentation/java/manual.md
Comment (cartermp) [inline content/en/docs/instrumentation/java/manual.md:388]:
```suggestion
//...
```

File: content/en/docs/instrumentation/java/manual.md
Comment (cartermp) [inline content/en/docs/instrumentation/java/manual.md:354]: Intentation here is much larger than the previous code samples, could you adjust it so that it's consistent?

File: content/en/docs/instrumentation/java/manual.md
Comment (cartermp) [inline content/en/docs/instrumentation/java/manual.md:351]: Hmmm. I don't really understand this sentence. What does "fetching the incoming context propagated" mean?

**Review by cartermp — CHANGES_REQUESTED** [review]

File: content/en/docs/instrumentation/java/manual.md
Comment (punidramesh) [reply content/en/docs/instrumentation/java/manual.md:351]: What I was trying to convey was that the trace contains some preexisting spans

File: content/en/docs/instrumentation/java/manual.md
Comment (punidramesh) [reply content/en/docs/instrumentation/java/manual.md:412]: Yep, the outer scope does not require a `finally` block as the span ends within the inner scope.

**Review by cartermp — APPROVED** [review]
Excellent, thank you! I think the changes look good. I would want to have one of the Java approvers also sign off on the code sample before merging.

**Review by jkwatson — APPROVED** [review]

**Pull request merged**
